packages/ui/src/stories/molecules/Tables/Accordion.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { AccordionContent } from '../../../components';
```

Comment:
The story name should be the same as the component name. In this case, it should be `AccordionContent.stories.ts`